grammars/php.cson

```diff
@@ -314,6 +314,41 @@
         ]
       }
       {
+        'begin': '(?x)\n\t\t\t\t\t\t\t\\s*(array) # Typehint\n\t\t\t\t\t\t\t\\s*(&)? \t\t\t\t\t# Reference\n\t\t\t\t\t\t\t\\s*((\\$+)[a-zA-Z_\\x{7f}-\\x{ff}][a-zA-Z0-9_\\x{7f}-\\x{ff}]*) # The variable name\n\t\t\t\t\t\t\t\\s*(=)\t# A default value\n\t\t\t\t\t\t\t\\s*()\\s*(\\\[)\n\t\t\t\t\t\t\t'
+        'beginCaptures':
+          '1':
+            'name': 'storage.type.php'
+          '2':
+            'name': 'storage.modifier.reference.php'
+          '3':
+            'name': 'variable.other.php'
+          '4':
+            'name': 'punctuation.definition.variable.php'
+          '5':
```

Comment:
It should not be included, since short array don't have `array` before `[]` I just leave it empty. It can be changed to `(?:)` but when removing the fifth row the `=` get the `array` type color. Should i change it to `(?:)`?